packages/http/src/config.ts

```diff
@@ -36,9 +36,9 @@ export function init(value: TConfig): void {
 
 export function getConfig(): TConfig {
   return {
-    apiPublicKey: assertNonEmptyString(config.apiPublicKey, "apiPublicKey"),
-    apiPrivateKey: assertNonEmptyString(config.apiPrivateKey, "apiPrivateKey"),
-    baseUrl: assertNonEmptyString(config.baseUrl, "baseUrl"),
+    apiPublicKey: config.apiPublicKey || "",
+    apiPrivateKey: config.apiPrivateKey || "",
+    baseUrl: config.baseUrl || "https://coordinator-beta.turnkey.io",
```

Comment:
let's add the `assertNonEmptyString` back? Seems useful.

I'm also not a fan of hardcoding `https://coordinator-beta.turnkey.io` at the core of our SDK. We may change domains or subdomains at some point.

Reply:
Fair enough. I'll figure out a good pattern for initializing this in the browser